test.html

```diff
@@ -1,162 +1,207 @@
+              <li><h4>Company</h4></li>
+              <li><a href="">About</a></li>
+              <li><a href="">Careers</a></li>
+              <br />
+              <li><a href="">Support</a></li>
+              <br />
+              <li><a href="">Media kit</a></li>
+              <br />
+              <li><a href="">Testimonials</a></li>
+              <br />
+              <li><a href="">API</a></li>
+              <br />
+              <li><a href="">Terms of service</a></li>
+              <br />
+              <li><a href="">Privacy policy</a></li>
+              <br />
+            </ul>
+            <ul>
+              <li><h4>Directories</h4></li>
+              <li><a href="">Design jobs</a></li>
+              <li><a href="">Designers for hire</a></li>
+              <br />
+              <li><a href="">Freelance designers for hire</a></li>
+              <br />
+              <li><a href="">Tags</a></li>
+              <br />
+              <li><a href="">Places</a></li>
+              <li><h4>Design assets</h4></li>
+              <li><a href="">Creative Market</a></li>
+              <li><a href="">Fontspring</a></li>
+              <br />
+              <li><a href="">Font Squirrel</a></li>
+              <br />
+            </ul>
+            <ul>
+              <li><h4>Design Resources</h4></li>
+              <li><a href="">Freelancing</a></li>
+              <li><a href="">Design Hiring</a></li>
+              <br />
+              <li><a href="">Design Portfolio</a></li>
+              <br />
+              <li><a href="">Design Education</a></li>
+              <br />
+              <li><a href="">Creative Process</a></li>
+              <br />
+              <li><a href="">Design Industry Trends</a></li>
+              <br />
+            </ul>
+          </div>
+        </div>
+        <div class="pg1_box8">
+          <h4>© 2022 Dribbble. All rights reserved.</h4>
+          <h5>17,903,686 shots dribbbled</h5>
+        </div>
+      </div>
     </section>
-    
-</body>
-<script src="https://kit.fontawesome.com/7ebee56059.js" crossorigin="anonymous"></script>
-<script>
-    var mlink = document.getElementById("menuLink");
-    function menuon(){
-        mlink.style.right = "0px";
+  </body>
+  <script src="https://kit.fontawesome.com/7ebee56059.js" crossorigin="anonymous"></script>
+  <script>
+    const mlink = document.getElementById('menuLink');
+    const openMenu = document.querySelector('#open-menu');
+    const closeMenu = document.querySelector('#close-menu');
+
+    function menuOn() {
+      //   mlink.style.right = '0px';
+      mlink.style.display = 'block';
     }
-    function menuoff(){
-        mlink.style.right = "-270px";
+
+    function menuOff() {
+      //   mlink.style.right = '-270px';
+      mlink.style.display = 'none';
     }
-</script>
-</html>
\ No newline at end of file
+
+    openMenu.addEventListener('click', menuOn);
+    closeMenu.addEventListener('click', menuOff);
+  </script>
```

Comment:
Instead of using HTML events, use JS. I've removed the animation transition as it was pushing the div off screen leading to a horizontal scroll bar to the page which isn't visually ideal. The way to approach this is to have a container div with **overflow: hidden** and then the menu should slide in and out of it so it acts like a mask through which you see the menu.